modules/logging-loki-alerts.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+:_content-type: PROCEDURE
+[id="logging-loki-alerts_{context}"]
+= Enabling log based alerts with Loki
+
+Loki alerting rules use link:https://grafana.com/docs/loki/latest/logql/[LogQL] and follow link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus formatting]. You can set log based alerts by creating an `AlertingRule` custom resource (CR). `AleringRule` CRs may be created for `application`, `audit`, or `infrastructure` tenants.
+
+[options="header"]
+|================================================
+| Tenant type    | Valid namespaces
+| application    |
+| audit          | `openshift-logging`
+| infrastructure | `openshift-/*`, `kube-/*`, `default`
+|================================================
+
+Application, Audit and Infrastructure alerts are sent to the Cluster Monitoring Operator (CMO) Alertmanager in the `openshift-monitoring` namespace by default, unless you have disabled the local `Alertmanager` instance.
+
+Application alerts are not sent to the CMO Alertmanager in the `user-workload-monitoring` namespace by default, unless you have enabled a separate `Alertmanager` instance.
```

Comment:
We can link at the end of this sentence to this guide: https://docs.openshift.com/container-platform/4.12/monitoring/enabling-alert-routing-for-user-defined-projects.html#enabling-a-separate-alertmanager-instance-for-user-defined-alert-routing_enabling-alert-routing-for-user-defined-projects

Reply:
That is easier said than done, I'm going to submit this for review as-is since this isn't coming out immediately and make sure to add it before requesting a merge.

Reply:
Completed.

Reply:
Are we planning on adding a section for RulerConfig?
https://github.com/openshift/openshift-docs/pull/57371#issuecomment-1514125011